app/views/polls/index.html.haml

```diff
@@ -2,15 +2,25 @@
 %table
   %tbody
     %tr
-      Open Polls:
+      Date Started:
+    %tr
+      Currently active:
+    %tr
+      Winner:
     -@polls.each do |poll|
       %tr
         %td
           #{ poll.created_at }
         %td
+          #{ !poll.closed }
+        %td
+          #{ poll.winner.try(:brand) }
```

Comment:
I use try here because if a poll hasn't been finalized yet it's winner is nil therefore I can't access `winner.brand` yet